src/Confluent.RestClient/ConfluentClient.cs

```diff
@@ -163,20 +165,11 @@ private HttpRequestMessage CreateRequestMessage(HttpMethod method, string reques
 
             return new HttpRequestMessage(method, requestUri);
         }
-
+       
         private async Task<ConfluentResponse<TResponse>> SendRequest<TResponse>(HttpRequestMessage request)
```

Comment:
Let's rename this method to `ProcessRequest` (I know I wrote it though :)) It seems this method does sending request + serialising.

`ProcessRequest` seems rather a best fit, and it won't affect anything as it's a private method.